packages/agoric-cli/src/commands/inter.js

```diff
@@ -66,7 +88,42 @@ const fmtMetrics = (metrics, quote, assets) => {
 };
 
 /**
- * Format amounts etc. in a bid OfferStatus
+ * Dynamic check that an OfferStatus is also a BidSpec.
+ *
+ * @param {import('@agoric/smart-wallet/src/offers.js').OfferStatus} offerStatus
+ * @param {Awaited<ReturnType<import('../lib/rpc').makeAgoricNames>>} agoricNames
+ * @param {typeof console.warn} warn
+ * returns null if offerStatus is not a BidSpec
+ */
+const coerceBid = (offerStatus, agoricNames, warn) => {
+  const { offerArgs } = offerStatus;
+  /** @type {unknown} */
```

Comment:
without a declaration, it would be treated as `any` and I'd get no help from `tsc`. This way, if I make any assumptions about `collateralBrand`, `tsc` checks them for me.